stories/tools/Listing.stories.js

```diff
@@ -11,6 +11,7 @@ import {
 import constants from "../../src/constants";
 
 const toolListingUriRegexp = /\/api\/tools.*/;
+const adminToolListingUriRegexp = /\/api\/admin\/tools.*/;
```

Comment:
I wonder if this would work:
```suggestion
const toolListingUriRegexp = /\/api\/(admin\/)?tools.*/;
```